src/CardArea/CardArea.js

```diff
@@ -61,12 +61,21 @@ class CardArea extends Component {
   }
   
   expandDeck = (deckObj) => {
-    const selectedDeck = this.props.getExpandedDeckInfo(deckObj)
-    this.setState({ selectedDeck });
+    const cardsInDeck = this.props.getExpandedDeckInfo(deckObj);
+    cardsInDeck.map(cardInDeck => {
+      const matchedCard = this.props.userCardsData.find(userCard => {
+        return userCard.cardName === cardInDeck.cardName;
+      });
+      cardInDeck.requiredCount = deckObj.cardCounts[cardInDeck.cardName];
+      cardInDeck.userCount = matchedCard ? 
+        Math.min(cardInDeck.requiredCount, matchedCard.cardCount) : 0;
```

Comment:
I used a ternary here instead of an if-else statement because the if-else added a bunch of lines and I didn't want to make the method unnecessarily long. The ternary assigns `userCount` to 0 if the `matchedCard` is null/falsey.

Also I used Math.min so that the `userCount` wouldn't be greater than the `requiredCount`.

Reply:
I like the ternary here especially with the math.min! very nice work.  We should look to see if there are other places we might be able to cut out our if elses.

Reply:
The ternary is nicely used here!